src/pages/overlay/components/overlay/settings/Settings.tsx

```diff
@@ -22,7 +22,7 @@ export default function Settings(props: OverlayOptionProps) {
 
           return (
             <li key={key}>
-              {setting.type === "boolean" && (
+              {setting.type === "boolean" && setting.configurable && (
```

Comment:
Instead of adding this check here, can we do it before the return, like the dev-only check?